website/mails/mail_callbacks.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, timedelta
+
+from website import settings
+from modularodm import Q
+from modularodm.exceptions import NoResultsFound, MultipleResultsFound
+
+def no_addon(email):
+    if len(email.user.get_addons()) is 0 and email.user.is_registered:
+        return True
+
+def no_login(email):
+    if not email.user.is_registered or email.user.date_last_login > datetime.utcnow() - settings.NO_LOGIN_WAIT_TIME:
```

Comment:
I think you can just do `return email.user.is_registered or email.user.date_last_login > datetime.utcnow() - settings.NO_LOGIN_WAIT_TIME`